OWNERS

```diff
@@ -1,11 +1,23 @@
 # See the OWNERS docs: <https://git.k8s.io/community/contributors/guide/owners.md>
 
 approvers:
-- maysunfaisal
+- elsony
 - johnmcollier
-- mike-hoang
+- kim-tsao
+- maysunfaisal
+- michael-valdron
+- thepetk
+- yangcao77
+- feloy
+- rm3l
```

Comment:
Do we need all these folks to be on the approvers and reviewers list?

Reply:
https://github.com/devfile/alizer/pull/25#discussion_r1323252267